packages/async-loader/async.js

```diff
@@ -1,36 +1,99 @@
-import { h, Component } from 'preact';
+import { h, Component, options, render } from 'preact';
 
-export default function(req) {
-	function Async() {
-		Component.call(this);
+const DOM = '__e';
+const CHILDREN = '__k';
+const PARENT = '__';
+const UNMOUNT = 'unmount';
+const oldUnmountOpts = options[UNMOUNT];
+const oldDiffed = options.diffed;
+const IS_PRERENDERED =
+	document.querySelector('#app') &&
+	document.querySelector('#app').childElementCount;
```

Comment:
should this get casted to a bool?

Reply:
like `!!(document.querySelector('#app') &&document.querySelector('#app').childElementCount);`

Reply:
I changed this